credsweeper/file_handler/data_content_provider.py

```diff
@@ -42,6 +45,47 @@ def data(self, data: bytes) -> None:
         """data setter"""
         self.__data = data
 
+    def represent_as_structure(self) -> bool:
+        """Tries to convert data with many parsers. Stores result to internal structure
+        Return True if some structure found
+        """
+        try:
+            text = self.data.decode(encoding='utf-8', errors='strict')
+        except Exception:
+            return False
+        # JSON
+        try:
+            if "{" in text:
+                self.structure = json.loads(text)
+                logger.debug("CONVERTED from json")
+                # logger.debug("CONVERTED from '%s' json:\n%s", self.data.decode(encoding='utf-8', errors='strict'),
+                #              str(self.structure))
```

Comment:
Could you please unify the logging style with `YAML` code block(L75-L76)?
And please remove annotated codes.